CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.1)
+
+# Install gtest which now lives on head. 
```

Comment:
Usually in cmake you do something like `find_package(GTest required)` to locate a library but ofc google doesn't do that now so they want to download a zip and unpack it. This is the magic to do that.